hammr/utils/deployment_utils.py

```diff
@@ -233,4 +274,32 @@ def show_deploy_progress_openstack(image_object, deployed_instance_id, bar_statu
     bar_status.message = "Deployment complete"
     progress.update(bar_status.percentage)
     progress.finish()
-    return status
\ No newline at end of file
+    return status
+
+def query_password_azure(question):
```

Comment:
Could be a generic method to ask password (remove "azure" from method name)

Reply:
At first I made a generic method, but inside the method I used a regexp to check the password, and it is specific to azure. So I think it should stay specific to azure, what do you think ?

Reply:
ok